js/content.js

```diff
@@ -16,6 +16,9 @@ let video = document.createElement("video");
 // creates checkmark element container (div)
 let checkElement = document.createElement("div");
 
+// creates spinner
+let spinnerDiv = document.createElement("div");
```

Comment:
Because the spinner needs to have an id associated with it, could this be created in the `addSpinner` method? The spinner could be removed by its id.